### PR TITLE
Fix doc printer missing [] around tuple type

### PR DESCRIPTION
**Pull request by SyrupThinker**
`deno doc` generates the following incorrect documentation for [this file](https://doc.deno.land/https/gist.github.com/SyrupThinker/24a9d41d4369a883a972b3b0be6b34cd/raw/3e594b86130a509d6410d9f925078fafc9679336/test.ts).
```
function one(i: Generic<number, string>): Generic<number, string>

function two(i: GenericTwo<number, string>): GenericTwo<number, string>

interface Generic

interface GenericTwo
```

With the tuple type printing fixed it now generates this documentation:
```
function one(i: Generic<[number, string]>): Generic<[number, string]>

function two(i: GenericTwo<number, string>): GenericTwo<number, string>

interface Generic

interface GenericTwo
```

### CODE REVIEW

**Comment by ry** [comment]
Thanks @SyrupThinker! can you add a test?

File: cli/tests/integration_tests.rs [hunk 1]
```diff
@@ -45,6 +45,35 @@ fn std_lint() {
...
+fn doc_tests() {
+  for file in util::root_path()
+    .join("cli/tests/doc")
+    .as_path()
+    .read_dir()
+    .unwrap()
+  {
+    let file = file.unwrap();
+    assert!(file.file_type().unwrap().is_file());
+
```
Comment (bartlomieju) [inline cli/tests/integration_tests.rs:58]: @SyrupThinker there's really no need to add integration tests for this PR. You could add new test case in `cli/doc/tests.rs`:

```rust
 assert!(
    colors::strip_ansi_codes(super::printer::format(entries).as_str())
      .contains("Generic<[string, number]>")
```

File: test_util/src/lib.rs [hunk 2]
```diff
@@ -191,16 +191,16 @@ pub fn run_python_script(script: &str) {
...
-pub struct CheckOutputIntegrationTest {
-  pub args: &'static str,
-  pub output: &'static str,
-  pub input: Option<&'static str>,
-  pub output_str: Option<&'static str>,
+pub struct CheckOutputIntegrationTest<'a> {
+  pub args: &'a str,
+  pub output: &'a str,
+  pub input: Option<&'a str>,
+  pub output_str: Option<&'a str>,
```
Comment (bartlomieju) [inline test_util/src/lib.rs:198]: Please revert, unrelated change

Comment (SyrupThinker) [reply test_util/src/lib.rs:198]: It is related to the commit already requested to be reverted ;)
